cmd/podman/images/scp.go

```diff
@@ -53,32 +60,48 @@ func init() {
 
 func scpFlags(cmd *cobra.Command) {
 	flags := cmd.Flags()
-	flags.BoolVarP(&scpOpts.Save.Quiet, "quiet", "q", false, "Suppress the output")
+	flags.BoolVarP(&source.Quiet, "quiet", "q", false, "Suppress the output")
 }
 
 func scp(cmd *cobra.Command, args []string) (finalErr error) {
 	var (
 		// TODO add tag support for images
 		err error
 	)
-	if scpOpts.Save.Quiet { // set quiet for both load and save
-		scpOpts.Load.Quiet = true
+	for i, val := range os.Args {
+		if val == "image" {
+			break
+		} else if i == 0 {
+			continue
+		} else if strings.Contains(val, "CIRRUS") {
+			continue
+		}
+		parentFlags = append(parentFlags, val)
+	}
+	podman, err := os.Executable()
+	if err != nil {
+		return err
 	}
 	f, err := ioutil.TempFile("", "podman") // open temp file for load/save output
 	if err != nil {
 		return err
 	}
-	defer os.Remove(f.Name())
-
-	scpOpts.Save.Output = f.Name()
-	scpOpts.Load.Input = scpOpts.Save.Output
-	if err := parse.ValidateFileName(saveOpts.Output); err != nil {
+	source.File = f.Name()
+	dest.File = source.File
+	err = os.Remove(source.File) // remove the file and simply use its name so podman creates the file upon save. avoids umask errors
+	if err != nil {
 		return err
 	}
+	_, err = os.Open(source.File)
```

Comment:
nit: do this on a single line
if err = os.Open
Does this leak a file descriptor?  Or is Golang smart enough to close the file. Why are you opening it, and not doing anything with the file descriptor?

Reply:
This is a remnant of a previous implementation, I think I can remove this.